extensions.json

```
@@ -283,6 +283,21 @@
       "id": "zhuangtongfa.material-theme",
       "version": "3.2.4",
       "repository": "https://github.com/Binaryify/OneDark-Pro"
+    },
+    {
+      "id": "ms-vscode.node-debug",
+      "version": "1.35.3",
+      "repository": "https://github.com/microsoft/vscode-node-debug"
+    },
+    {
+      "id": "ms-vscode.node-debug2",
+      "version": "1.33.0",
```

Latest available version seems to be `1.42.1`: https://github.com/microsoft/vscode-node-debug2/releases

```suggestion
      "version": "1.42.1",
```

---

See https://github.com/open-vsx/publish-extensions/pull/16#issuecomment-603300080